test/git-https-use-case.js

```diff
@@ -14,6 +14,9 @@ test('npm-shrinkwrap --dev on git+https uri', fixtures(__dirname, {
         'package.json': JSON.stringify({
             name: 'foo',
             version: '0.1.0',
+            description: "",
```

Comment:
I add these to avoid nom warning messages pop up during tests, warning like this:
```
npm WARN package.json foo@0.1.0 No description
npm WARN package.json foo@0.1.0 No repository field.
npm WARN package.json foo@0.1.0 No README data
```

If that's not a problem, I could revert these changes.